doc/manual/install-judgehost.rst

```diff
@@ -50,15 +50,31 @@ home directory::
   make judgehost
   sudo make install-judgehost
 
+For the next section we assume a machine with possibly hyperthreading and more
+than 3 CPUs. This can be checked with::
+
+  lscpu | grep "Thread(s) per core"
+
+having a value above 1 indicates hyperthreading or::
+
+  cat /sys/devices/system/cpu/smt/active
+
+a value of `1` or `on`. The target CPU for the next session should be in the range of::
+
+  cat /sys/devices/system/cpu/online
+
+if the machine has less than 3 cores and has no hyperthreading change the value to `1`.
```

Comment:
Nitpick: this is a new sentence, so should start with a capitalized `If`.
Better to explain in more detail here how to possibly pick a CPU core and why we picked `2` as default. For example if the machine has only a single core then `1` will obviously not work.